entrypoint.sh

```diff
@@ -73,6 +62,7 @@ cat <<EOF
     "buildNumber": ${BUILDNUMBER},
     "branchName": "${BRANCHNAME}",
     "commitHash": "${COMMITHASH}",
+    "kafkaTopics": ${KAFKA_TOPICS_JSON},
```

Comment:
Yes, we need that. Now we consider less if-cases, and feed the kafkaTopics into the trigger-request json irrespective of whether it was filled from the file or not. In case the file is empty, and empty json array is the value of "kafkaTopics". See line 36.